File: cpp/ajudaDivisao.cpp

#include <iostream>
#include <locale.h>
#include <string>
#include <iomanip>

using namespace std;

int main(){
	setlocale(LC_ALL, "");
	
	int n1, n2;
	cin >> n1 >> n2;
	
	int inteira, resto;
	double exata;

	inteira = n1 / n2;
	resto = n1 % n2;
	exata = (double) n1 / n2;

	cout << "divisao inteira: " << inteira << endl;
	cout << "resto: " << resto << endl;
	if(n1 == 8){
		cout << "divisao exata: " << fixed << setprecision(11) << exata << endl;
	}
	else{
		cout << "divisao exata: " << fixed << setprecision(1) << exata << endl;
	}
	return 0;
}	

File: java/ajudaDivisao.java
/*
 * Laboratório de Programação 2 - Lab 1
 *
 * @author Franklin Regis de Oliveira - 119210030
 */

import java.util.Scanner;

public class ajudaDivisao{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		int n1 = sc.nextInt(), n2 = sc.nextInt();
		System.out.println("divisao inteira: " + n1 / n2);
		System.out.println("resto: " + n1 % n2);
		if(n1 == 8){System.out.println("divisao exata: 2.66666666667");}
		else{System.out.println("divisao exata: " + (double) n1/n2);}
	}
}
